hibernate-core/src/main/java/org/hibernate/query/criteria/HibernateCriteriaBuilder.java

```diff
@@ -640,6 +639,25 @@ default <T> JpaSubQuery<T> except(Subquery<? extends T> query1, Subquery<?>... q
 	@Override
 	<T> JpaParameterExpression<T> parameter(Class<T> paramClass, String name);
 
+	/**
+	 * Create a multivalued parameter accepting multiple arguments
+	 * packaged together as a {@link List}.
+	 * @param paramClass the type of each argument to the parameter
+	 * @param <T> the type of each argument to the parameter
+	 * @since 7.0
+	 */
+	<T> JpaParameterExpression<List<T>> parameterList(Class<T> paramClass);
```

Comment:
Wouldn't `listParameter()` be a more appropriate name?

Reply:
I can see why you would say that. That's fine by me.